docs/table/construct_table.rst

```diff
@@ -201,7 +201,7 @@ column where each row element is itself a two-element array.
     ...
   KeyError: 'a_new'
 
-Row Data
+List of Rows
 --------
```

Comment:
The line below needs to match in length the line above. Silly but that's the convention.

```suggestion
------------
```